tests/test_crud.py

```diff
@@ -88,6 +88,9 @@ def test_crud(tmpdir, module, vcrmode):
         run = run_playbook_vcr(tmpdir, module, record=record)
     assert run.rc == 0
 
+    for event in run.events:
+        assert not event.get('event_data', {}).get('warning', False)
```

Comment:
```suggestion
        assert not event.get('event_data', {}).get('warning', False), str(event.event_data.warning)
```
Let's see the warning.

Reply:
It's a bool, so there is nothing to see, but it's also the wrong bool. It'll only show you play warnings, I want task warnings, and that's what I've implemented now.